### DISCUSSION TITLE
TypeError: replaceAll is not a function

### DISCUSSION THREAD

**Discussion by vwcity** [qutebrowser/qutebrowser#7454] [Q&A] [answered]
Hi,
On a recent website I have this error "TypeError: replaceAll is not a function" on javascript file.
Do you have a solution ?
Thank's

**Comment by toofar** [comment, marked as answer]
It depends on what qutebrowser version you have, what QtWebEngine version you are running it against and if you can upgrade. 
If you are on a QtWebEngine version lower than 5.15.3 and a qutebrowser version 2.2.0 or greater (you can check by running `:version`) we ship a polyfill for String.replaceAll but it's disabled by default. You can remove the `js-string-replaceall` value from the `content.site_specific_quirks.skip` setting (eg `:set content.site_specific_quirks.skip ''`) and then restart to enable it.
Or see if you can get a newer QtWebEngine version.

**Comment by vwcity** [comment]
I'm on qutebrowser version 2.5.2 with QtWebEngine version 5.15.2, I can upgrade QtWebEngine ? (I'm on Windows)

**Reply by toofar** [reply]
Not easily, no. Qt doesn't distribute free binary builds of Qt 5.15.3+, nor do other reputable sources so I guess they don't allow distributing them in general.

**Comment by vwcity** [comment]
It's works with `:set content.site_specific_quirks.skip ''`
Thanks !!
